packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -88,6 +91,75 @@ class MethodWebsocketRequestHandler {
     }
   }
 
+  MethodStreamManager _createMethodStreamManager(
+    WebSocket webSocket,
+    Server server,
+  ) {
+    return MethodStreamManager(
+      onInputStreamClosed: (
+        UuidValue namespace,
+        String parameterName,
+        CloseReason? closeReason,
```

Comment:
I took a bit of a shortcut here since the enum is very generic and it also simplifies the usage in `MethodWebsocketRequestHandler`.  Strictly speaking we should duplicate it, but it will clutter the code more so I opted to just reuse it. Let know if you think we should duplicate it!